aiosql/queries.py

```diff
@@ -54,6 +58,10 @@ def fn(self, conn, *args, **kwargs):
                 return self.driver_adapter.select_one(
                     conn, query_name, sql, parameters, record_class
                 )
+            elif operation_type == SQLOperationType.SELECT_VALUE:
```

Comment:
sorry for the duplication :) I should find a better way to do this for both wrapper functions.